src/domain/priceGroup/priceGroupForm/PriceGroupForm.tsx

```diff
@@ -0,0 +1,247 @@
+  const locale = useLocale();
+  const navigate = useNavigate();
+  const { user } = useUser();
+
+  const action = priceGroup
+    ? PRICE_GROUP_ACTIONS.UPDATE
+    : PRICE_GROUP_ACTIONS.CREATE;
+  const savedPlacePublisher = getValue(priceGroup?.publisher, '');
+
+  const { organizationAncestors } =
+    useOrganizationAncestors(savedPlacePublisher);
+
+  const { saving, updatePriceGroup } = usePriceGroupActions({ priceGroup });
+
+  const { serverErrorItems, setServerErrorItems, showServerErrors } =
+    usePriceGroupServerErrors();
+
+  const goToPriceGroupsPage = () => {
+    navigate(`/${locale}${ROUTES.PRICE_GROUPS}`);
+  };
+
+  const onUpdate = async (values: PriceGroupFormFields) => {
+    await updatePriceGroup(values, {
+      onError: (error: ServerError) => showServerErrors({ error }),
+      onSuccess: () => {
+        goToPriceGroupsPage();
+        addNotification({
+          label: t('priceGroup.form.notificationPriceGroupUpdated'),
+          type: 'success',
+        });
+      },
+    });
+  };
+
+  const isEditingAllowed = checkCanUserDoPriceGroupAction({
+    action,
+    organizationAncestors,
+    publisher: savedPlacePublisher,
+    user,
+  });
+
+  const inputRowBorderStyle = useMemo(
+    () =>
+      /* istanbul ignore next */
+      isEditingAllowed ? '' : styles.borderInMobile,
+    [isEditingAllowed]
+  );
+
+  const alignedInputStyle = useMemo(
+    () =>
+      /* istanbul ignore next */
+      isEditingAllowed
+        ? styles.alignedInput
+        : styles.alignedInputWithFullBorder,
+    [isEditingAllowed]
+  );
+
+  return (
+    <Formik
+      enableReinitialize={true}
+      initialValues={
+        priceGroup
+          ? getPriceGroupInitialValues(priceGroup)
+          : PRICE_GROUP_INITIAL_VALUES
+      }
+      // We have custom way to handle onSubmit so here is empty function
+      // to silent TypeScript error. The reason for custom onSubmit is that
+      // we want to scroll to first invalid field if error occurs
+
+      onSubmit={
+        /* istanbul ignore next */
+        () => undefined
+      }
+      validateOnMount
+      validateOnBlur={true}
+      validateOnChange={true}
+      validationSchema={isEditingAllowed && priceGroupSchema}
+    >
+      {({ setErrors, setTouched, values }) => {
+        const clearErrors = () => setErrors({});
+
+        const publisher = priceGroup ? savedPlacePublisher : values.publisher;
+
+        const disabledIfEditing = !isEditingAllowed || !!priceGroup;
+
+        const handleSubmit = async (
+          event?: React.FormEvent<HTMLFormElement>
+        ) => {
+          event?.preventDefault();
+
+          try {
+            setServerErrorItems([]);
+            clearErrors();
+
+            await priceGroupSchema.validate(values, { abortEarly: false });
+
+            if (priceGroup) {
+              await onUpdate(values);
+            } else {
+              // await onCreate(values);
+            }
```

Comment:
```suggestion
            }
```
Since this is coming in PR 3/4, it's best to not have anything in this PR/commit here IMO.

Reply:
Removed the extra else statement